src/wmrc/main.py

```diff
@@ -302,7 +302,7 @@ def _update_facilities(self, gis: arcgis.gis.GIS, facility_summary_df: pd.DataFr
                 "latitude",
                 "longitude",
                 "tons_of_material_diverted_from_",
-                "gallons_of_used_oil_collected_for_recycling_last_year",
+                "gallons_of_used_oil_collected_f",
```

Comment:
**question**: should we add a data cleaning function to palletjack to warn about or alter long column names?

Reply:
This was a weird situation where it really shouldn't have been long in the first place. Generally AGOL doesn't allow the longer file names, but this long name snuck in somehow and stuck until the manual update. 

palletjack already verifies that the incoming fields match the existing AGOL fields; this fix is the result of that check.